packages/fast-development-site-react/src/components/design-system/index.ts

```diff
@@ -2,7 +2,12 @@ export interface DevSiteDesignSystem {
     foregroundColor: string;
     backgroundColor: string;
     brandColor: string;
-    lightGray: string;
+    background000: string;
+    background100: string;
+    background200: string;
+    background300: string;
+    background800: string;
+    foreground300: string;
```

Comment:
Since we are setting defaults, we should make these optional so that we don't break anyone.